client/my-sites/pages/page/index.js

```diff
@@ -736,6 +737,11 @@ class Page extends Component {
 		this.props.recordEvent( 'Clicked Delete Page' );
 	};
 
+	recordPageTitleClick = () => {
+		this.props.recordTracksEvent( 'calypso_pages_page_edit_click', { page_type: 'real' } );
+		this.props.recordEvent( 'Clicked Page Title' );
```

Comment:
Instead of calling `this.props.recordEvent( 'Clicked Page Title' );` here, I would call `this.props.recordPageTitle()`

Reply:
Thanks